### ISSUE TITLE
Blank admin page after install magento 2.3.0 on Windows 10

### ISSUE THREAD

**Issue by iamdylanngo** [magento/magento2#20084]
### Preconditions (*)

1. On Windows 10
2. Blank admin page

### Steps to reproduce (*)

1. Install magento 2.3.0 complete on Windows 10
2. Go to url admin page

### Expected result (*)

1. Go to page admin success
2. Login to admin

### Actual result (*)

1. Blank admin page
![screenshot_1](https://user-images.githubusercontent.com/11944757/50723046-e7558f00-110a-11e9-9bf0-610b47b9ec42.png)
![screenshot_2](https://user-images.githubusercontent.com/11944757/50723047-eb81ac80-110a-11e9-9f2d-ce713d2a8fa5.png)

### Environment 
![screenshot_3](https://user-images.githubusercontent.com/11944757/50723138-3819b780-110c-11e9-989c-575f5a60f9d6.png)
![screenshot_4](https://user-images.githubusercontent.com/11944757/50723140-394ae480-110c-11e9-99c4-34965af8c441.png)

Looking forward to your feedback 👍

**Comment by magento-engcom-team** [comment]
Hi @jundat95. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@jundat95 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by orlangur** [comment]
Closing as duplicate of https://github.com/magento/magento2/issues/19589. Please check that older ticket for solutions.
